### PR TITLE
movie: Simone (2002) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Riddled with plot holes. Rating 1/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
...
+famous.
+
+:: Date
+2002-08-23
+
+:: References
+https://en.wikipedia.org/wiki/Simone_(2002_film)
+
+:: Ratings
+1 <paul>
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2540]: An outrage. It's Al Pacino! Al Pacino!

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
...
+
+:: Major Themes
+the desire for prestige [The story follows the fading film director Viktor Taransky as he takes the desperate measure of secretly inserting a virtual actress into his films in an effort to get the recognition he feels that he deserves. In Taransky's own words, "I wanted recognition" he said in regard to his motives for perpetrating this hoax.]
+coping with personal failure [The fading film director Viktor Taransky resorted to an elaborate hoax in an effort to change his fortunes after having suffered ten years of "abject failure in the film industry".]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2544]: https://www.themeontology.org/theme.php?name=coping%20with%20being%20washed%20up ?

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 3]
```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
...
+filmmaking [Viktor Taransky's using of a virtual actress in his films was a main novelty of the story.]
+old flames [The divorcees Viktor and Elaine rekindled their relationship over the course of the film, and in the end they got back together.]
+father and daughter [Viktor was advised by his precocious young daughter Lainey about this career and relationships over the course of the film, and he tried as he might to show he cared for her.]
+criminal fraud [Viktor Taransky clandestine use of a virtual actress in his hit films was tantamount to fraud.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2548]: Was it? Justify!

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 4]
```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
...
+hoaxes in society [The fading film director Victor Taransky fooled the world into thinking the virtual actress who was starring in his hit films was a real woman, and he took elaborate measures to keep her virtual nature a secret as pressure from the media for her to make public appearances mounted.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2549]: I sense a slippery slope here.

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 5]
```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
...
+celebrity worship [The whole world instantly came to adore the new star actress Simone - the only thing the public didn't know was that she was nothing more than a piece of software.]
+speculative technology [A main novelty of the film is the virtual actress Simone - a non-sentient piece of software that when presented on screen was indistinguishable to the public from a real actress.]
+sociotechnological issue [The viewer is made to ponder about the ramifications on society of using software to simulate such realistic likeness of human beings that the general public mistakes them for the real articles.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:2552]: Consider introducing a "deep fake" kind of theme some day soon.

**Review by odinlake — COMMENTED** [review]
Notes inline.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:2544]: Good call. Added.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:2540]: A chef is only as good as his ingredients.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:2548]: Knocked down to minor with new motivation.

**Review by odinlake — APPROVED** [review]

**Pull request merged**